microshift_install/microshift-install-rpm.adoc

```diff
@@ -16,27 +16,27 @@ Red Hat does not support an update path from Developer Preview and Technology Pr
 include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
 include::modules/microshift-install-rpm-preparing.adoc[leveloffset=+1]
 
-[role="_additional-resources_microshift-install-rpm"]
+[role="_additional-resources"]
+[id="_additional-resources_microshift-install-rpm"]
 .Additional resources
 
-* xref:../microshift_install/microshift-install-rpm.adoc#system-requirements-installing-microshift[System Requirements for Installing MicroShift] have been met.
-
 * Download the link:https://console.redhat.com/openshift/install/pull-secret[pull secret] from the Red Hat Hybrid Cloud Console.
 
-* xref:../microshift_configuring/microshift-using-config-tools.adoc#microshift-using-config-tools[Configuring MicroShift].
+* link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12/html/configuring/microshift-using-config-tools[Configuring MicroShift].
```

Comment:
Not sure if you want to switch this one or not to an xref.

Reply:
Pushing...